lib/l10n/intl_en.arb

```
@@ -767,6 +797,58 @@
+  "@failedSendMessage": {
+    "placeholders": {
+      "error": { "type": "String" }
+    }
+  },
+  "waitingAdminAssignmentInput": "Waiting for admin assignment...",
+  "typeYourMessage": "Type your message...",
+  
+  "@_comment_dispute_list": "Dispute List Text",
+  "disputesWillAppear": "Disputes will appear here when you open them from your trades",
+  "failedLoadDisputes": "Failed to load disputes",
+  "retry": "Retry",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)